src/pages/index.hbs

```diff
@@ -1,6 +1,6 @@
 ---
 layout: src/templates/layouts/landing.hbs
-title: An Invitation to Joy
+title: Ceaseless Prayer - An Invitation to Joy
```

Comment:
Did you not want to change this? By the way, this is for the landing page too since it's a template.